Source/Particles/ElementaryProcess/QEDInternals/BreitWheelerEngineWrapper.cpp

```diff
@@ -61,7 +61,7 @@ bool BreitWheelerEngine::are_lookup_tables_initialized () const
 bool
 BreitWheelerEngine::init_lookup_tables_from_raw_data (
     const vector<char>& raw_data,
-    const amrex::Real bw_minimum_chi_phot)
+    const amrex::ParticleReal bw_minimum_chi_phot)
```

Comment:
```suggestion
    const amrex::Real bw_minimum_chi_phot)
```

Keep this as `Real`?